frontend/www/js/omegaup/components/contest/NewForm.vue

```diff
@@ -320,14 +355,39 @@ export default class NewForm extends Vue {
     this.penalty = 20;
     this.penaltyType = 'none';
     this.showScoreboardAfter = true;
+    this.contestStyleDesc = T.contestNewFormTitlePlaceholderConacupStyle;
   }
 
   onSubmit() {
-    this.$emit('emit-update-contest', this);
-  }
-
-  mounted() {
-    $('.selectpicker', this.$el).selectpicker();
+    if (this.update) {
+      this.$emit('emit-update-contest', this);
+      return;
+    }
+    this.$emit('create-contest', {
+      alias: this.alias,
+      title: this.title,
+      description: this.description,
+      start_time: this.startTime
+        ? this.startTime.getTime() / 1000
+        : this.initialStartTime.getTime() / 1000,
+      finish_time: this.finishTime
+        ? this.finishTime.getTime() / 1000
+        : this.initialFinishTime.getTime() / 1000,
+      window_length:
+        this.windowLength === null || !this.windowLengthEnabled
+          ? 0
+          : this.windowLength,
+      points_decay_factor: this.pointsDecayFactor,
+      submissions_gap: this.submissionsGap ? this.submissionsGap * 60 : 60,
+      languages: this.languages,
+      feedback: this.feedback,
+      penalty: this.penalty,
+      scoreboard: this.scoreboard,
+      penalty_type: this.penaltyType,
+      show_scoreboard_after: this.showScoreboardAfter,
+      basic_information: this.needsBasicInformation ? 1 : 0,
```

Comment:
esto no debería ser booleano?

```suggestion
      basic_information: this.needsBasicInformation,
```

Reply:
Cierto